proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -613,6 +613,8 @@ message SyncActivityRequest {
     google.protobuf.Timestamp last_attempt_complete_time = 19;
     // Stamp represents the internal “version” of the activity options and can/will be changed with Activity API.
     int32 stamp = 20;
+    // Indicates if the activity is paused.
+    bool paused = 21;
```

Comment:
How about we store `pause_info` with a struct instead of a boolean? It'll let us surface more information at a glance, such as who initiated pausing and when.

Reply:
no at this iteration (will be in batch activity API)